packages/orbit-components/src/Select/Select.stories.tsx

```diff
@@ -16,237 +17,230 @@ const objectOptions = [
-    options={objectOptions}
-    error={<div>You need to select some value.</div>}
-    onChange={action("onChange")}
-  />
-);
-
-WithErrorMessage.story = {
-  name: "With error message",
+export const WithHelpMessage: Story = {
+  args: {
+    label: "Select with help message",
+    help: "Most common choice is Booking cancellation",
+    helpClosable: true,
+  },
 
   parameters: {
-    info: "Selects are used for showing content hierarchy and are important for improving the reading experience for our users. Visit Orbit.Kiwi for more detailed guidelines.",
+    info: "Select component with help message tooltip. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onChange", "onBlur", "onFocus", "placeholder", "error", "inlineLabel"],
+    },
   },
 };
 
-export const Playground = ({
-  placeholder,
-  disabled,
-  name,
-  customValueText,
-  option,
-  value,
-  dataTest,
-  spaceAfter,
-  id,
-  required,
-  dataAttrs,
-  width,
-  label,
-  inlineLabel,
-  error,
-  help,
-}) => {
-  return (
-    <Select
-      id={id}
-      required={required}
-      placeholder={placeholder}
-      width={width}
-      options={option}
-      disabled={disabled}
-      name={name}
-      label={label}
-      inlineLabel={inlineLabel}
-      onChange={action("onChange")}
-      onBlur={action("onBlur")}
-      onFocus={action("onFocus")}
-      dataTest={dataTest}
-      value={value}
-      customValueText={customValueText}
-      spaceAfter={spaceAfter}
-      dataAttrs={dataAttrs}
-      error={error}
-      help={help}
-    />
-  );
-};
+export const WithErrorMessage: Story = {
+  args: {
+    label: "Select with error message",
+    error: "You need to select some value.",
+  },
 
-Playground.story = {
   parameters: {
-    info: "Selects are used for showing content hierarchy and are important for improving the reading experience for our users. Visit Orbit.Kiwi for more detailed guidelines.",
+    info: "Select component with error message tooltip. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onChange", "onBlur", "onFocus", "placeholder", "help", "inlineLabel"],
+    },
   },
 };
 
-Playground.args = {
-  placeholder: "Select value from list",
-  disabled: false,
-  name: "name",
-  customValueText: "",
-  option: objectOptions,
-  value: undefined,
-  dataTest: "test",
-  spaceAfter: SPACINGS_AFTER.SMALL,
-  id: "select-id",
-  required: false,
-  dataAttrs: { "data-recording-ignore": true },
-  width: "",
-  label: "Label",
-  inlineLabel: false,
-  error: "",
-  help: "",
-};
+export const Playground: Story = {
+  render: ({ icon, ...args }) => {
+    const Icon = typeof icon === "string" && getIcon(icon);
 
-Playground.argTypes = {
-  spaceAfter: {
-    options: Object.values(SPACINGS_AFTER),
-    control: {
-      type: "select",
+    return <Select {...args} prefix={<Icon />} />;
+  },
+
+  args: {
+    ...WithPrefixAndLabel.args,
+    label: "Select",
```

Comment:
`label` is already specified in `WithPrefixAndLabel.args` so it's unnecessary here, no?